frontend/public/components/routes.tsx

```diff
@@ -94,9 +39,15 @@ export const getRouteWebURL = (route: RouteKind): string => {
   return url;
 };
 
-const getSubdomain = (route: RouteKind): string => {
-  const hostname = _.get(route, 'spec.host', '');
-  return hostname.replace(/^[a-z0-9]([-a-z0-9]*[a-z0-9])\./, '');
+export const RouteLinkAndCopy: React.FC<RouteLinkAndCopyProps> = ({ route, className }) => {
```

Comment:
Move the exported methods to utils folder or elsewhere, and delete this file in a follow-on PR.